docs/content/doc/installation/from-binary.en-us.md

```diff
@@ -24,13 +24,38 @@ embedded assets. This can be different for older releases.
 
 ## Download
 
-Choose the file matching your platform from the [downloads page](https://dl.gitea.io/gitea/), copy the URL and replace the URL within the commands below:
+You can find the file matching your platform from the [downloads page](https://dl.gitea.io/gitea/) after navigating to the version you want to download. 
+
+### Choosing the right file
+For Linux, you should choose: 
+ * `linux-386` for 32-bit Intel/AMD processors
+ * `linux-amd64` for 64-bit Intel/AMD processors
+ * 'linux-arm64' for 64-bit ARM systems like a Raspberry PI 4
+ * `linux-arm-5` and `linux-arm-6` are for other versions of the ARM processor 
+ 
+For Windows, you should chose:
+ * `gogit-windows-4.0-386` for 32-bit Intel/AMD processors
+ * `gogit-windows-4.0-amd64` for 64-bit Intel/AMD processors
+
+The `gogit-windows` file addresses [significant performance problems](https://github.com/go-gitea/gitea/pull/15482) that have been reported on Windows by including a built-in, Go-based git client for some operations. The `windows`-only file (i.e. without `gogit`) will exclusively use whichever git is client installed on the server.
```

Comment:
I think we should be careful not to over-egg this. The main slow downs were probably related to the earlier iterations of the git code. 

Some users may find that the go-git variant is much faster for them depending on how their git installation and how their windows systems tolerate spawning processes.

It would useful to know if all windows users gain benefit from go-git or if it is just some.